### ISSUE TITLE
Override formatData.js

### ISSUE THREAD

**Issue by yarikwest** [excl-networks/strapi-plugin-ezforms#18]
Hi,
how can I override formatData function? I was trying plugins extension from strapi docs [https://docs.strapi.io/developer-docs/latest/development/plugins-extension.html#within-the-extensions-folder](url) like this:
```
module.exports = (plugin) => {
  plugin.services.email.send = (ctx) => {
    console.log('!!email send!!')
  };
  return plugin;
};
```
Unfortunately it doesn't work.

I use strapi 4.3.6 and ezforms 0.1.0

**Comment by Cpaczek** [comment]
Right now formatData isn't registered as a Strapi service so there wouldn't be a way to override it, its probably best to move it to the services folder, (probably move the entire utils folder there) I will get around to it soon or you're free to open a PR.

**Comment by Cpaczek** [comment]
Hi,
This has been implemented in v0.1.1 please update and the plugin and you should be able to override the `formatData` function as its now a service

**Comment by Cpaczek** [comment]
@yarikwest This has been added to docs https://ezforms.excl.dev/advanced/custom-formatting.html
